Hometask8/Car/Car.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Random;
+
+public class Car {
+    private String carBrend;
+    private int speed;
+    private int cost;
+
+    public Car() {
+    }
+
+    public Car(String carBrend, int speed, int cost) {
+        this.carBrend = carBrend;
+        this.speed = speed;
+        this.cost = cost;
+    }
+
+    public String getCarBrend() {
+        return carBrend;
+    }
+
+    public void setCarBrend(String carBrend) {
+        this.carBrend = carBrend;
+    }
+
+    public int getSpeed() {
+        return speed;
+    }
+
+    public void setSpeed(int speed) {
+        this.speed = speed;
+    }
+
+    public int getCost() {
+        return cost;
+    }
+
+    public void setCost(int cost) {
+        this.cost = cost;
+    }
+
+    class EvenNumberException extends Exception {
+        public EvenNumberException(String name) {
+            System.err.println("Машина " + name + " сломалась");
```

Comment:
лучше здесь не печатать а передать это сообщение в supper
```suggestion
            super("Машина " + name + " сломалась");
```